### PR TITLE
GA sdk release.

### PR DESCRIPTION
**Pull request by wangbwn**

### CODE REVIEW

**Review by ruowan — APPROVED** [review]

**Review by jsquire — DISMISSED** [review]
@wangbwn :

Your account lacks write permissions to the repository, which is required before this PR can move forward. Please review the Azure SDK onboarding documentation and use the associated teams channel for assistance.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by ruowan** [comment]
updated code owner file. https://github.com/Azure/azure-sdk-for-net/pull/42345/files

File: sdk/datafactory/Azure.ResourceManager.DataFactory/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 
 ### Features Added
 
+- Improve Salesforce V2 properties.
+- Add ServiceNowV2 linkedService, dataSet.
+- Add GoogleBigQuery linkedService, dataSet.
+- Add PostgreSqlV2 linkedService, dataSet.
 - Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
 
 ### Breaking Changes
```
Comment (ruowan) [inline sdk/datafactory/Azure.ResourceManager.DataFactory/CHANGELOG.md:13]: Please remove breaking changes. And other changes seem it cause validation failure.

**Comment by ruowan** [comment]
@JoshLove-msft We are releasing the .NET GA version SDK. The CI failure report that
`eng\Directory.Build.Common.targets(316,5): Error : When the project has a release version it shouldn't reference any pre-release libraries. Found the following pre-release references: Azure.Core.Expressions.DataFactory`   Seems we need to release a Azure.Core.Expression.DataFactory first and then release the ADF GA version .net SDK.  Could you please help to take a look?

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Pending on Josh release Azure.Core.Expressions.DataFactory first stable

**Review by jsquire — DISMISSED** [review]
`wangbwn` still requires write permissions to the repository for this PR to move forward.

**Comment by wangbwn** [comment]
Hi, @JoshLove-msft.
"I locally modified the version of Azure.Core.Expressions.DataFactory to '1.0.0', then regenerated the code. However, I found that the 'dotnet build /t:GenerateCode' command did not rename DataFactorySecretBaseDefinition to DataFactorySecret as expected."
image

File: sdk/openai/Azure.AI.OpenAI.Assistants/api/Azure.AI.OpenAI.Assistants.netstandard2.0.cs [hunk 2]
```diff
@@ -263,6 +263,74 @@ public partial class FunctionToolDefinition : Azure.AI.OpenAI.Assistants.ToolDef
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.AI.OpenAI.Assistants.FunctionToolDefinition>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct InternalAssistantDeletionStatusObject : System.IEquatable<Azure.AI.OpenAI.Assistants.InternalAssistantDeletionStatusObject>
```
Comment (ruowan) [inline sdk/openai/Azure.AI.OpenAI.Assistants/api/Azure.AI.OpenAI.Assistants.netstandard2.0.cs:266]: Pls don't update this file openai.

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/datafactory/Azure.ResourceManager.DataFactory/src/autorest.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ generate-model-factory: false
 csharp: true
```
Comment (ArthurMa1978) [inline sdk/datafactory/Azure.ResourceManager.DataFactory/src/autorest.md:9]: Need confirm why `generate-model-factory: false` which disable generate model factory

**Pull request merged**